service/service.go

```diff
@@ -137,7 +137,7 @@ func (s *Service) CreateUser(ctx context.Context, req *pb.UserRequest) (*pb.User
 		return nil, status.Error(codes.Internal, err.Error())
 	}
 
-	logger.Info("Inserted new user:", user.GetUuid())
+	logger.Info("Inserted new user:", user.GetUuid(), user.GetFirstName(), user.GetLastName())
 
 	return &pb.UserResponse{
 		Status:  &pb.UserResponse_Code{Code: uint32(codes.OK)},
```

Comment:
According to README.md,
Returns the created User with password field set to empty string

Reply:
`insertNewUser` needs not to mutate as well